public/common/components/timer/view-models/timer.viewmodel.coffee

```diff
@@ -0,0 +1,43 @@
+{ isNumeric } = require '../../../utils/data-type-utils'
+
+class TimerViewModel
+	constructor: (remainingCell, format) ->
+		@remaining = new nx.Cell
+			'<<-': [ remainingCell, @selectFormatter(format) ]
```

Comment:
Immediate binding. Find more here: https://github.com/DaQuirm/nexus/wiki/Bindings-and-Cells#immediate-sync
The idea is to draw the timer immediately. So if there's any value (but timer is not running yet) - the time will be rendered but not an empty string. Also it was a bit easier to write tests :8ball: